src/pool/srv_target.c

```diff
@@ -1253,6 +1253,8 @@ ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 	ABT_rwlock_unlock(pool->sp_lock);
 	if (map != NULL)
 		pool_map_decref(map);
+	if (rc == 0)
+		rc = ds_cont_rf_check(pool->sp_uuid);
```

Comment:
One more question I'm struggling with: Is it possible that

    ~ xstream 0
        pool map changes from 10 to 11 and breaks RF for container X
        ds_cont_rf_check()
            wait for eventual
    ~ xstream 1
        receives a read/write to container X with pool map 11
        replies OK
    ~ xstreams
        check RFs and set the new rw disable flags
    ~ xstream 0
        ds_cont_rf_check returns 0

Reply:
I thought it again, two related things may consider -
1) node/target failure cannot be notified to all nodes at same time "atomically" - different node may set the rw disable flag at different time.
2) on same node, we even cannot ensure all VOS XS set the rw disable flag at the same time atomically. We use ULT collective to set each XS' container child, still a possible time window that different XS with different status.

TBH, I am not very sure if that is a problem/concern to us, maybe others can comment here.
I'll leave this one to be addressed later, thanks.

Reply:
Fine with me to leave this to future work.